app/main.py

```diff
@@ -1 +1,44 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    try:
+        mv_command, source_file, destination = command.split()
+    except ValueError:
+        raise ValueError(
```

Comment:
It’s a bad practice to catch exception and then raise this again. You can just print message, it’s  a good catch

Reply:
Thank you! Just changed it. Is other ValueError raising is fine?